Userland/Libraries/LibWeb/Layout/FlexFormattingContext.cpp

```diff
@@ -90,9 +90,9 @@ void FlexFormattingContext::run(Box const& run_box, LayoutMode, AvailableSpace c
 
     // 3. Determine the flex base size and hypothetical main size of each item
     for (auto& item : m_flex_items) {
-        if (item.box->is_replaced_box()) {
+        if (box_is_sized_as_replaced_element(*item.box)){ //or should it be item.box->is_sized_as_replaced_element()?
```

Comment:
Comment obsolete

Reply:
Does this include "// FIXME: This should not be done *by* ReplacedBox"?

Reply:
I assume this doesn't include "// FIXME: Get rid of prepare_for_replaced_layout() and make replaced elements figure out their intrinsic size lazily."

Reply:
Yeah